tests/providers/google/cloud/transfers/test_mysql_to_gcs_system.py

```diff
@@ -0,0 +1,81 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+import pytest
+from psycopg2 import ProgrammingError
+
+from airflow.providers.mysql.hooks.mysql import MySqlHook
+from airflow.providers.google.cloud.example_dags.example_mysql_to_gcs import GCS_BUCKET
+from tests.providers.google.cloud.utils.gcp_authenticator import GCP_GCS_KEY
+from tests.test_utils.gcp_system_helpers import CLOUD_DAG_FOLDER, GoogleSystemTest, provide_gcp_context
+
+CREATE_QUERY = """
+CREATE TABLE test_table
+(
+    id int auto_increment primary key,
+    params json
+);
+"""
+
+LOAD_QUERY = """
+INSERT INTO test_table (id, params)
+VALUES
+   (
+      1, '{ "customer": "Lily Bush", "items": {"product": "Diaper","qty": 24}}'
+   ),
+   (
+      2, '{ "customer": "Josh William", "items": {"product": "Toy Car","qty": 1}}'
+   ),
+   (
+      3, '{ "customer": "Mary Clark", "items": {"product": "Toy Train","qty": 2}}'
+   );
+"""
+DELETE_QUERY = "DROP TABLE test_table;"
+
+
+@pytest.mark.backend("mysql")
+@pytest.mark.credential_file(GCP_GCS_KEY)
+class MySQLToGCSSystemTest(GoogleSystemTest):
+    @staticmethod
+    def init_db():
+        try:
+            hook = MySqlHook()
+            hook.run(CREATE_QUERY)
+            hook.run(LOAD_QUERY)
+        except ProgrammingError:
```

Comment:
```suggestion
        except (OperationalError, ProgrammingError):
```

```
query = b'\nCREATE TABLE test_table\n(\n    id int auto_increment primary key,\n    params json\n);\n'

    def query(self, query):
        # Since _mysql releases GIL while querying, we need immutable buffer.
        if isinstance(query, bytearray):
            query = bytes(query)
        if self.waiter is not None:
            self.send_query(query)
            self.waiter(self.fileno())
            self.read_query_result()
        else:
>           _mysql.connection.query(self, query)
E           _mysql_exceptions.OperationalError: (1050, "Table 'test_table' already exists")

/usr/local/lib/python3.7/site-packages/MySQLdb/connections.py:280: OperationalError
```